iot_hub.py

```diff
@@ -0,0 +1,64 @@
+import os
+import time
+from pin import Pin
+from azure.iot.device import IoTHubDeviceClient
+
+conn_str = os.getenv("IOTHUB_DEVICE_CONNECTION_STRING")
+device_client = IoTHubDeviceClient.create_from_connection_string(conn_str)
+
+PIN_NUMBER = '11'  # REPLACE WITH YOUR PIN NUMBER if different
+configured_pin = Pin(PIN_NUMBER)
+
+# Connect the IoTHub client
+device_client.connect()
+
+# Get the device twin document
+twin = device_client.get_twin()
+print("Initial twin document:")
+print("{}".format(twin))
+
+
+def report_light_state(state):
+    reported_properties = {"light": state}
+    light_state = reported_properties["light"]
+    print(f"Setting reported light state to {light_state}")
+    device_client.patch_twin_reported_properties(reported_properties)
+
+
+# Initalize the light as off in reported properties:
+report_light_state("OFF")
+
+
+while True:
+    request = input('Enter ON, OFF, LISTEN, or CLEAR: ')
+    if request == "CLEAR":
```

Comment:
Same as for the other one. CTRL+C and handling SIGINT might even be enough

Reply:
Same as above - I'll handle with a finally, I think that should handle the SIGINT by deafault.